app/helpers/stream_helper.rb

```diff
@@ -29,6 +21,24 @@ def reshare?(post)
 
   private
 
+  def next_stream_path
+    if current_page?(:stream)
+      stream_path(max_time: time_for_scroll(@stream))
```

Comment:
- Assignment Branch Condition size for next_stream_path is too high. [21.84/20]
- Cyclomatic complexity for next_stream_path is too high. [7/6]
- Perceived complexity for next_stream_path is too high. [8/7]